remix-lib/src/execution/txRunner.js

```diff
@@ -214,27 +214,27 @@ class TxRunner {
   }
 }
 
-async function tryTillReceiptAvailable (txhash, done) {
+async function tryTillReceiptAvailable (txhash, executionContext) {
   return new Promise((resolve, reject) => {
```

Comment:
Would you like to remove this Promise class as `async` function will return promise anyway.

Reply:
not sure that is really possible in this context: the body of the function contain also an async function...

Reply:
can that be?

Reply:
hmmm, may be we can try and know better.